app/src/main/res/layout/fragment_speaker.xml

```diff
@@ -5,11 +5,60 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
+    <Button
+        android:id="@+id/btnSetting_Speaker"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Setting"
```

Comment:
It's better to specify null instead of transparent background color.
```suggestion
        android:text="Setting"
        android:background="@null"
```